### PR TITLE
Repeat layer for column vector

### PR DESCRIPTION
**Pull request by emailweixu**

### CODE REVIEW

File: python/paddle/trainer_config_helpers/layers.py [hunk 1]
```diff
@@ -2846,17 +2861,19 @@ def seq_concat_layer(a, b, act=None, name=None, layer_attr=None,
...
+
+    Note that the above computation is for one sample. Multiple samples are
+    processed in one batch.
 
     The example usage is:
 
     ..  code-block:: python
 
-        concat = seq_concat_layer(a=layer1, b=layer2)
+        concat = seq_concat_layer(al=layer1, b=layer2)
```
Comment (qingqing01) [inline python/paddle/trainer_config_helpers/layers.py]: `al` -> `a`

File: paddle/gserver/layers/FeatureMapExpandLayer.cpp [hunk 2]
```diff
@@ -62,6 +63,7 @@ bool FeatureMapExpandLayer::init(const LayerMap& layerMap,
 
   CHECK_EQ(inputLayers_.size(), 1UL);
   numFilters_ = config_.num_filters();
+  asRowVector_ = config_.user_arg() != "as_col_vec";
```
Comment (qingqing01) [inline paddle/gserver/layers/FeatureMapExpandLayer.cpp]: `user_arg` had been defined in [ModelConfig.proto](https://github.com/PaddlePaddle/Paddle/blob/develop/proto/ModelConfig.proto#L441).

File: python/paddle/trainer_config_helpers/layers.py [hunk 3]
```diff
@@ -1566,13 +1566,21 @@ def expand_layer(input,
 
 @wrap_name_default()
 @layer_support()
-def repeat_layer(input, num_repeats, name=None, layer_attr=None):
+def repeat_layer(input,
+                 num_repeats,
+                 as_row_vector=True,
+                 name=None,
+                 layer_attr=None):
```
Comment (qingqing01) [inline python/paddle/trainer_config_helpers/layers.py]: The activation is implemented in C++ code. Should we add it to the Python interface?

File: python/paddle/trainer_config_helpers/layers.py (hunk 1 above)
Comment (emailweixu) [inline python/paddle/trainer_config_helpers/layers.py]: Done

File: python/paddle/trainer_config_helpers/layers.py (hunk 3 above)
Comment (emailweixu) [inline python/paddle/trainer_config_helpers/layers.py]: Done

**Comment by qingqing01** [comment]
The `test_repeat_layer.py` was omitted.

```
Generating  test_repeat_layer
Traceback (most recent call last):
  File "/opt/python/2.7.12/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/python/2.7.12/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/travis/build/PaddlePaddle/Paddle/python/paddle/utils/dump_config.py", line 25, in <module>
    conf = parse_config(sys.argv[1], '')
  File "/home/travis/build/PaddlePaddle/Paddle/python/paddle/trainer/config_parser.py", line 3596, in parse_config
    make_config_environment(trainer_config, config_args))
IOError: [Errno 2] No such file or directory: 'test_repeat_layer.py'
```

**Pull request merged**